leap_ec/probe.py

```diff
@@ -394,6 +418,8 @@ class AttributesCSVProbe(op.Operator):
         as a list of individuals, and their return value is printed in the column.
     :param int job: a job ID that will be included as a constant-value column in
         all rows (ex. typically an integer, indicating the ith run out of many)
+    :param bool numpy_as_json: if True, numpy arrays will be first converted to
```

Comment:
Those would be the numpy arrays for the genomes, so mentioning that would be helpful, especially to those folks new to LEAP.